### PR TITLE
Generalized insert method insert(name, doc)

### PR DESCRIPTION
**Pull request by arkilic**

### CODE REVIEW

**Comment by arkilic** [comment]
NOT ready to merge yet @danielballan

File: metadataclient/commands.py [hunk 1]
```diff
@@ -985,9 +988,13 @@ def _normalize_human_friendly_time(val):
...
 
-# TODO: Add when descriptor is None, use cached descriptor!
-# TODO: Fix descriptor replace (related to above)
-# TODO: Add server_disconnect that rolls all config back to default
-# TODO: Add capped collection caching layer for the client
-# TODO: Add fast read/write capped collection, different than caching capped collection
-# TODO: Add timeouts to servers in order to stop ppl from abusing data sources
+ins_dict = {'run_start': insert_run_start, 'run_stop': insert_run_stop,
+            'descriptor': insert_descriptor, 'event': insert_event,
+            'bulk_event': bulk_insert_events}
```
Comment (ericdill) [inline metadataclient/commands.py]: To be consistent with the RunEngine, consider using the values in the `event_model.DocumentNames` enum: https://github.com/NSLS-II/event-model/blob/master/event_model/__init__.py#L9

Comment (ericdill) [inline metadataclient/commands.py]: Err using the values in the DocumentNames enum as the keys for `ins_dict`

Comment (danielballan) [inline metadataclient/commands.py]: That would add a dependency that MDC does not currently have, so strings are fine by me.

**Pull request merged**